packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -893,3 +926,55 @@ class _TooltipOverlay extends StatelessWidget {
     );
   }
 }
+
+/// A controller to manage visibility of a tooltip.
+///
+/// A [TooltipController] is used to control and interrogate a tooltip after it
+/// has been created, with methods such as [show] and [conceal],
+/// and state accessors like [isOpen].
+///
+/// See also:
+///
+/// * [Tooltip], a material design tooltip.
+///   [TooltipController].
+class TooltipController {
+  /// The tooltip that this controller controls.
+  ///
+  /// This is set automatically when a [TooltipController] is given to the
+  /// tooltip it controls.
+  TooltipState? _state;
+
+  /// Whether or not the associated tooltip is currently open.
+  bool get isOpen {
```

Comment:
When would the developer want to use `isOpen`?  Also if the tooltip is fading in/out I'm not sure what `isOpen` is supposed to indicate?

Reply:
> When would the developer want to use `isOpen`? Also if the tooltip is fading in/out I'm not sure what `isOpen` is supposed to indicate?

You are right. I was thinking of a situation in which user wants to open another tooltip when the other one is already opened/closed. But as you mentioned that tooltips are fading in/out there isOpen would be useless.